source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,364 @@
+  this.route('contact');
+  this.route('rentals', function() {
+    this.route('show');
+  });
+});
+```
+
+Our new route is nested within our `rentals` route.
+This tells Ember that it is a sub-route and will be accessed through `localhost:4200/rentals/show`.
+
+To tell the application which rental we want to access, we need to replace the `show` route path with the ID of the rental listing.
+We also want to simplify the URL so that it looks more like this: `localhost:4200/rentals/id-for-rental`.
+
+To do that, we modify our route as follows:
+
+```app/router.js{+5}
+Router.map(function() {
+  this.route('about');
+  this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:rental_id' });
+  });
+});
+```
+
+The `rental_id` will now be passed to the route.
+
+
+## Finding By ID
+
+Next, we want to edit our `show` route to retrieve the requested rental:
+
+```app/routes/rentals/show.js{+2,+3,+4}
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.findRecord('rental', params.rental_id);
+  }
+});
+```
+
+Since we added `:rental_id` to the `show` path in our router, `rental_id` is now available in our `model` hook.
+When we call `this.store.findRecord('rental', params.rental_id)`, Ember Data queries `/rentals/our-id` using a HTTP GET request ([learn more about that here](../../models/)).
+
+## Adding the Rental To Our Template
+
+Next, we can update the template for our show route (`app/templates/rentals/show.hbs`) and list the information for our rental.
+
+```app/templates/rentals/show.hbs
+<div class="jumbo show-listing">
+  <h2 class="title">{{model.title}}</h2>
+  <div class="right detail-section">
+    <div class="detail owner">
+      <strong>Owner:</strong> {{model.owner}}
+    </div>
+    <div class="detail">
+      <strong>Type:</strong> {{rental-property-type model.type}} - {{model.type}}
+    </div>
+    <div class="detail">
+      <strong>Location:</strong> {{model.city}}
+    </div>
+    <div class="detail">
+      <strong>Number of bedrooms:</strong> {{model.bedrooms}}
+    </div>
+    <p>{{model.description}}</p>
+  </div>
+  <img src="{{model.image}}" class="rental-pic">
+</div>
+```
+
+Now browse to `localhost:4200/rentals/grand-old-mansion` and you should see the information listed for that specific rental.
+
+![Rental Page Nested Show Route](../../images/subroutes/subroutes-super-rentals-show.png)
+
+## Linking to a Specific Rental
+
+Now that we can load pages for individual rentals, we'll add a link (using the `link-to` helper) within our `rental-listing` component to navigate to individual pages.
+Here, the `link-to` helper takes the route name and the rental model object as arguments.
+When you pass an object as second argument into the `link-to` block helper, it will by default [serialize](http://emberjs.com/api/classes/Ember.Route.html#method_serialize) the object to the ID of the model into the URL. 
+Alternately, you may just pass `rental.id` for clarity.
```

Comment:
@acorncom I put some explanation around the use of rental in the link-to.  I like that fact that it exposes that route serialization is a thing.